amundsen_application/static/js/ducks/user/types.ts

```diff
@@ -52,3 +57,42 @@ export interface GetUserResponse {
   type: GetUser.SUCCESS | GetUser.FAILURE;
   payload?: User;
 }
+
+
+/* getUserOwn */
+export enum GetUserOwn {
+  REQUEST = 'amundsen/user/own/GET_REQUEST',
+  SUCCESS = 'amundsen/user/own/GET_SUCCESS',
+  FAILURE = 'amundsen/user/own/GET_FAILURE',
+}
+
+export interface GetUserOwnRequest {
+  type: GetUserOwn.REQUEST;
+  payload: {
```

Comment:
worthy mention: Seeing this reminded me that in my ducks refactor work I didn't consolidate the request actions that take params to use `payload` -- only did the response actions. Thanks for using it here. I'll clean that up for everything else.